examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
I think it is very unlikely that anyone goes to that example to understand the difference between None and nearest. I think the interpolation methods (accompanied by more text explanation) is more than enough to understand the difference.